orbit/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Orbit 1.27.0 (Jun 21, 2024)
+
+* Disabled `mdm_bridge` table on Windows Server.
+
+* Fixes an issue related to hardware UUIDs being cached in osquery's database. When an orbit install
+  is transferred from one machine to another (e.g. via MacOS Migration Assistant), the new machine
+  now shows up in Fleet as a separate host from the old one.
+
+* Added support for `--end-user-email` option when building fleetd Linux packages.
+
+* Fixed bug where MDM migration fails when attempting to renew enrollment profiles on macOS Sonoma devices.
+
+* Scripts are now enabled by default when fleet is automatically installed
```

Comment:
This line item still requires the download.fleetdm.com changes to be in effect so let's remove this until that is finished.
```suggestion
```

The work has been done that the arg is sent when enrolling but needs the updated fleetd-base.msi to be stored w/ this and referenced before it will work